phalcon/mvc/model.zep

```diff
@@ -1505,7 +1505,9 @@ abstract class Model implements EntityInterface, ModelInterface, ResultInterface
 				new Message(
 					message->getMessage(),
 					message->getField(),
-					message->getType()
+					message->getType(),
+					null,
```

Comment:
Im not really sure about this null, it should be this anyway, but this will break tests, not sure about apps).